frontend/public/components/overview/index.tsx

```diff
@@ -275,6 +276,32 @@ const sortBuilds = (builds: K8sResourceKind[]): K8sResourceKind[] => {
   return builds.sort(byBuildNumber);
 };
 
+const overviewEmptyStateToProps = ({UI}) => ({activeNamespace: UI.get('activeNamespace')});
+
+const OverviewEmptyState = connect(overviewEmptyStateToProps)(({activeNamespace}) => (
+  <EmptyState>
+    <EmptyState.Title>
+      Get started with your project.
+    </EmptyState.Title>
+    <EmptyState.Info>
+      Add content to your project from the catalog of web frameworks, databases, and other components. You may also deploy an existing image or create or replace resources using YAML and JSON definitions.
```

Comment:
Replace isn't supported.

```suggestion
      Add content to your project from the catalog of web frameworks, databases, and other components. You may also deploy an existing image or create resources using YAML definitions.
```